system_test/common/aest_channels_SUITE.erl

```diff
@@ -284,6 +286,10 @@ reestablish_state_channel_perform_operations({INodeName, RNodeName},
     ct:log("Testing mutual close"),
     {ok, CloseTxHash, _CloseFee, _IChange, _RChange} = sc_close_mutual(Chan1, initiator),
     wait_for_value({txs_on_chain, [CloseTxHash]}, NodeNames, 5000, Config),
+    %% wait for min depth to be reached so the channels die
```

Comment:
Channel FSM is alive until we the `channel_close_mutual_tx` is burried deep enough in the chain that we consider it fork-safe. Abruptly killing one FSM kills its noise handler which could result in error messages in the log of the other FSM, esp. if running older node version.